code/iaas/healthcheck/src/main/java/io/cattle/iaas/healthcheck/process/ServiceEventCreate.java

```diff
@@ -104,6 +111,19 @@ private boolean isNetworkUp(long accountId) {
                 return false;
             }
         }
+        if(networkDriverService.getState().equals(CommonStatesConstants.ACTIVE)) {
+            Object obj = DataAccessor.fields(networkDriverService).withKey(ServiceConstants.LAST_ACTIVE).get();
```

Comment:
there is a DataAccessor.FieldDate method that can be used

Reply:
It internally converts the long value to Date and returns null if key not present, I'll need to check for null and convert back to long anyway. 
Or am I understanding it wrong ?